### PR TITLE
Use the ActivityPub blog user as an actor if set to blog profile only

### LINKED ISSUE #410: ActivityPub: use the blog user to follow when only the blog profile is enabled
See https://wordpress.org/support/topic/activitypub-user-does-not-see-follow/

### PR DESCRIPTION
**Pull request by akirk**
Fixes #410

Instead of `$actor = \get_author_posts_url( $user_id );` we now use `$actor = \Activitypub\Collection\Actors::get_by_id( $user_id );` which also work for the ActivityPub blog actor.

### CODE REVIEW

**Comment by akirk** [comment]
@pfefferle is this a good way to do it?

File: feed-parsers/class-feed-parser-activitypub.php [hunk 1]
```diff
@@ -686,6 +686,21 @@ private function disable_polling( User_Feed $user_feed ) {
 		$user_feed->update_metadata( 'next-poll', gmdate( 'Y-m-d H:i:s', time() + YEAR_IN_SECONDS ) );
 	}
 
+	public function get_activitypub_actor( $user_id ) {
+		if ( null === $user_id ) {
+			$user_id = Friends::get_main_friend_user_id();
+			if ( class_exists( '\ActivityPub\Collection\Followers' ) ) {
```
Comment (pfefferle) [inline feed-parsers/class-feed-parser-activitypub.php:692]: is this to check if the ActivityPub plugin is installed? If so, I would suggest using `\define( 'ACTIVITYPUB_PLUGIN_VERSION' );` instead!

Comment (akirk) [reply feed-parsers/class-feed-parser-activitypub.php:692]: Ah, oops, I just reused some code from where I get the followers. Thanks.

**Comment by akirk** [comment]
@pfefferle my question was more whether it's ok to use such an `Activitypub\Collection\Actor` for `$activity->set_actor( $actor );`

**Pull request merged**